restapi/consts.go

```diff
@@ -18,16 +18,15 @@ package restapi
 
 const (
 	// consts for common configuration
-	ConsoleVersion               = `0.1.0`
-	ConsoleAccessKey             = "CONSOLE_ACCESS_KEY"
-	ConsoleSecretKey             = "CONSOLE_SECRET_KEY"
-	ConsoleMinIOServer           = "CONSOLE_MINIO_SERVER"
-	ConsoleMinIOServerTLSRootCAs = "CONSOLE_MINIO_SERVER_TLS_ROOT_CAS"
-	ConsoleProductionMode        = "CONSOLE_PRODUCTION_MODE"
-	ConsoleHostname              = "CONSOLE_HOSTNAME"
-	ConsolePort                  = "CONSOLE_PORT"
-	ConsoleTLSHostname           = "CONSOLE_TLS_HOSTNAME"
-	ConsoleTLSPort               = "CONSOLE_TLS_PORT"
+	ConsoleVersion        = `0.1.0`
```

Comment:
A breaking change so perhaps 

```suggestion
	ConsoleVersion        = `0.2.0`
```